packages/dev/core/src/FlowGraph/serialization.ts

```diff
@@ -63,15 +69,19 @@ export function defaultValueSerializationFunction(key: string, value: any, seria
  * @returns
  */
 export function defaultValueParseFunction(key: string, serializationObject: any, scene: Scene) {
-    const value = serializationObject[key];
+    const intermediateValue = serializationObject[key];
     let finalValue;
-    const className = value?.className;
+    const className = intermediateValue?.className;
     if (isMeshClassName(className)) {
-        finalValue = scene.getMeshByName(value.name);
+        finalValue = scene.getMeshByName(intermediateValue.name);
     } else if (isVectorClassName(className)) {
-        finalValue = parseVector(className, value.value);
+        finalValue = parseVector(className, intermediateValue.value);
+    } else if (className === "FGPath") {

```

Comment:
is this (`FGPath`) not a static member of the FlowGraphPath class?

Reply:
I hadn't added it as a static but I'll do that!